tutorials/algorithms/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,572 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pylab.rcParams[\"figure.figsize\"] = (12, 8)\n",
+    "pylab.plot(counts, values, label=type(optimizer).__name__)\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy\")\n",
+    "pylab.title(\"Energy convergence using Gradient\")\n",
+    "pylab.legend(loc=\"upper right\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Monitoring using logging\n",
+    "\n",
+    "Much of the code is instrumented with [Python logging](https://docs.python.org/3/library/logging.html) statements. The logging is configurable to adjust level of logging etc. Here we set the logging level to `INFO`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "# uncomment to set up logging\n",
+    "# logging.basicConfig(level=logging.INFO)\n",
+    "# logging.getLogger(\"qiskit.algorithms.minimum_eigen_solvers.vqe\").setLevel(logging.INFO)"
+   ]
+  },
```

Comment:
In the algorithms in Aqua we used to add more extensive logging than seems to get added nowadays. The example shown before would work and the logging below was a couple of lines copy/pasted from the output. As it stands the logging commented out above there is still enabling it for the old vqe not the new. BUT even if it were enabled the output as shown below (which is still from old VQE) would no longer occur as  I do not see there is logging in the energy evaluation function anymore. While there is logging there the only output now is minimal - in looking it logs when trying to resize an ansatz and at the end when the optimization is complete - so just 2 places. As such maybe we should just drop this entire logging section.

Reply:
Yes, that's true, I did not even try out the logging. Removing.